addon/templates/components/power-select-with-create.hbs

```diff
@@ -1,64 +1,66 @@
-{{#component powerSelectComponentName
-  afterOptionsComponent=afterOptionsComponent
-  allowClear=allowClear
-  ariaDescribedBy=ariaDescribedBy
-  ariaInvalid=ariaInvalid
-  ariaLabel=ariaLabel
-  ariaLabelledBy=ariaLabelledBy
-  beforeOptionsComponent=beforeOptionsComponent
-  buildSelection=buildSelection
-  calculatePosition=calculatePosition
-  class=class
-  closeOnSelect=closeOnSelect
-  defaultHighlighted=defaultHighlighted
-  destination=destination
-  dir=dir
-  disabled=disabled
-  dropdownClass=dropdownClass
-  extra=extra
-  groupComponent=groupComponent
-  highlightOnHover=highlightOnHover
-  horizontalPosition=horizontalPosition
-  initiallyOpened=initiallyOpened
-  loadingMessage=loadingMessage
-  matchTriggerWidth=matchTriggerWidth
-  matcher=matcher
-  noMatchesMessage=noMatchesMessage
-  onblur=onblur
-  onchange=(action selectOrCreate)
-  onclose=onclose
-  onfocus=onfocus
-  oninput=oninput
-  onkeydown=onkeydown
-  onopen=onopen
-  options=optionsArray
-  optionsComponent=optionsComponent
-  placeholder=placeholder
-  placeholderComponent=placeholderComponent
-  preventScroll=preventScroll
-  registerAPI=registerAPI
-  renderInPlace=renderInPlace
-  scrollTo=scrollTo
-  search=(action searchAndSuggest)
-  searchEnabled=searchEnabled
-  searchField=searchField
-  searchMessage=searchMessage
-  searchMessageComponent=searchMessageComponent
-  searchPlaceholder=searchPlaceholder
-  selected=selected
-  selectedItemComponent=selectedItemComponent
-  tabindex=tabindex
-  triggerClass=triggerClass
-  triggerComponent=triggerComponent
-  triggerId=triggerId
-  triggerRole=triggerRole
-  typeAheadMatcher=typeAheadMatcher
-  verticalPosition=verticalPosition
-  as |option term|
-}}
-  {{#if option.__isSuggestion__}}
-    {{component suggestedOptionComponent option=option term=term}}
-  {{else}}
-    {{yield option term}}
-  {{/if}}
-{{/component}}
+{{#let (component this.powerSelectComponentName) as |Select|}}
+  <Select
+    @afterOptionsComponent={{@afterOptionsComponent}}
+    @allowClear={{@allowClear}}
+    @ariaDescribedBy={{@ariaDescribedBy}}
+    @ariaInvalid={{@ariaInvalid}}
+    @ariaLabel={{@ariaLabel}}
+    @ariaLabelledBy={{@ariaLabelledBy}}
+    @beforeOptionsComponent={{this.beforeOptionsComponent}}
+    {{!-- @buildSelection={{@buildSelection}} --}}
```

Comment:
If you allow the user to overwrite `buildSelection` like that `@buildSelection={{@buildSelection}}`, but there is no `buildSelection` passed, the component falls back to `option => option` one, even with `<PowerSelectMultiple>`

Reply:
This should be fixed by https://github.com/cibernox/ember-power-select/pull/1271 as well.

Reply:
Looks like it is working after upgrading to 3.0.4